dvc/render/converter/vega.py

```diff
@@ -274,11 +274,9 @@ def flat_datapoints(self, revision):  # noqa: C901, PLR0912
             _update_all(
                 datapoints,
                 update_dict={
-                    VERSION_FIELD: {
```

Comment:
[F] I need to double check but I think the reason for having this inside of a nested dict was because we wanted to get rid of the data before it showed up in a tooltip. However, none of the tooltips from the updated templates will show any of this data. I am hoping it is safe to remove.